content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning metrics and other
-metadata in simple file formats, which is fully compatible with DVC.
+metadata in simple file formats which are fully compatible with DVC.
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code in order to start
+logging metrics.
```

Comment:
I do think this GS approach is better (straight into ML framework integration code sample). But do we still cover generic code application? I guess just in the API ref samples right? (Probably enough)

Reply:
> do we still cover generic code application? I guess just in the API ref 

I see it's in the [API index page](https://dvc-org-dvclive-refacto-czwjzz.herokuapp.com/doc/dvclive/api-reference) 👍🏼